lib/matplotlib/tests/test_text.py

```diff
@@ -30,7 +30,7 @@ def find_matplotlib_font(**kw):
     from matplotlib.font_manager import FontProperties, findfont
     warnings.filterwarnings(
         'ignore',
-        ('findfont: Font family \[u?\'Foo\'\] not found. Falling back to .'),
+        r'findfont: Font family \[u?\'Foo\'\] not found. Falling back to .',
```

Comment:
@Carreau As mentioned just above, can you switch to double quotes and remove the backslashes in the inner single quotes here?